libs/blocks/src/lib/search/search-block.stories.tsx

```diff
@@ -1,21 +1,17 @@
-import type { Meta, StoryObj } from '@storybook/react';
-import SearchBlock from '.';
+import { Meta, StoryObj } from '@storybook/react';
+import {
+  SearchResults,
+  SearchResultsProps,
+} from 'libs/templates/src/lib/help-center/sections/search/search-results';
```

Comment:
please don't import this directly here, just provide your own data and pass it to story. thanks 🙏 
this way we might confuse people that they can make circular deps in modules and libs.